services/device-registry-base/src/main/java/org/eclipse/hono/deviceregistry/service/deviceConnection/AutowiredDeviceConnectionAdapter.java

```diff
@@ -11,7 +11,7 @@
  * SPDX-License-Identifier: EPL-2.0
  *******************************************************************************/
 
-package org.eclipse.hono.deviceregistry;
+package org.eclipse.hono.deviceregistry.service.deviceConnection;
```

Comment:
Package name should be all lowercase, so `deviceconnection`.

Reply:
My bad. I have changed it now.